src/frontend/js/components/ReferralForm/TopicField.tsx

```diff
@@ -158,16 +171,47 @@ export const TopicField: React.FC<TopicFieldProps> = ({
           <TopicSuggestion {...{ topic, isHighlighted }} />
         )}
         shouldRenderSuggestions={() => true}
+        renderSuggestionsContainer={({ containerProps, children, query }) => (
+          <div {...containerProps}>
+            {children}
+
+            {!children &&
+              query.length > 0 &&
+              state.context.value.length === 0 &&
+              isInputFocused && (
+                <div>
+                  <FormattedMessage {...messages.noItemsfound} />
+                </div>
+              )}
+          </div>
+        )}
         inputProps={{
           id: seed('referral-topic-label'),
           'aria-describedby': seed('referral-topic-description'),
           placeholder: intl.formatMessage(messages.label),
           onChange: (_, { newValue }) => {
+            if (state.context.value.length > 0)
+              send({ type: 'CHANGE', data: '' });
+            else send('CLEAN');
+
             setValue(newValue);
           },
+          onFocus: () => {
+            setisInputFocused(true);
+          },
+          onBlur: () => {
+            setisInputFocused(false);
+          },
           value,
         }}
       />
+      {state.matches('cleaned.true') &&
+      state.matches('validation.invalid') &&
+      !isInputFocused ? (
```

Comment:
I think the logic here should be to show the message once clean Is true, no matter if the input is focused, until the user provides valid input.

If we do that, then we'll need to clean in `onBlur` as opposed to `onChange` 😉